src/containers/GeneAlleleDetailsPage/GeneAlleleDetailsTable.js

```diff
@@ -15,6 +15,34 @@ import { getSingleGenomeLocation, findFminFmax } from '../../lib/utils';
 import VariantsSequenceViewer from '../genePage/VariantsSequenceViewer';
 import ErrorBoundary from '../../components/errorBoundary';
 
+const getSiftStyle = (sift) => {
+  switch (sift && sift.toLowerCase()) {
+  case 'tolerated':
+    return 'text-success';
```

Comment:
The matching of predictions to a css is a bit weird. Rather than matching it to existing css classes that are used for success, warning and danger I would map it to new css classes with the color name or even better create classes such as 
text-sift-prediction-tolerated
text-sift-prediction-deleterious
etc.
That would be more explicit and does not tie any changes of text-danger, etc. with the colors of the predictions.

Reply:
`text-danger`, `text-warning` are bootstraps classes designed exactly for this purpose - to give meaning to a colour, ie red means dangerous, orange means warning. This also maps quite well to the different levels of severity to SIFT and PolyPhen prediction.

This is how the design system works. Each colour in the colour palette is chosen with a particular meaning to allow consistency across the site, so that you can change the colour theme in one place that would be reflected across the site. It's regular practice and not weird at all.

Reply:
In other words, the website should have a consistent way to communicate status to users through the consistent use of colour. What you suggest to avoid is actually desirable.

Reply:
What if we have more than three colors to use in our UI? If we are sure we only need danger, warning and success for the predictions then yes, we would map them to those three status flags.

Reply:
We can have more colours if situation calls for it. I think we are good here right? Is there anything else for this PR before you check it off?

The goal with the colours really is to help the user understand information better. But simply having more colours do not necessarily help users understand what they mean. This is why a colour legend is often used in visualizations.

We associate special meaning to red, orange, green. We learned what they means through how we interact with other UIs and in our everyday lives. So as designers, we can use those colour to communicate information and be reasonable certain it will be understood. If we use other colours, we need to be very deliberate and ensure the meaning of it communicated.